Lesson13_Real.html

```diff
@@ -0,0 +1,236 @@
+          tu 9
+          den 11!</p>
+      </div>
+      <div class="mb-3">
+        <label for="birthday" class="form-label">Birthday</label>
+        <input type="date" id="birthday" class="form-control" name="birthday">
+      </div>
+      <div class="mb-3" id="checkRadio">
+        <label for="sex" class="form-label">Gioi Tinh</label> <br>
+        <div class="form-check form-check-inline">
+          <input class="form-check-input" type="radio" name="inlineRadioOptions" id="inlineRadio1" value="Nam"
+            checked="checked">
+          <label class="form-check-label" for="inlineRadio1">Nam</label>
+        </div>
+        <div class="form-check form-check-inline">
+          <input class="form-check-input" type="radio" name="inlineRadioOptions" id="inlineRadio2" value="Nu">
+          <label class="form-check-label" for="inlineRadio2">Nu</label>
+        </div>
+      </div>
+      <div class="mb-3">
+        <label for="form-label" class="form-label">Address</label>
+        <input class="form-control" type="text" id="text-address" placeholder="Address">
+      </div>
+      <div class="mb-3">
+        <label for="info">Textarea Info</label>
+        <textarea class="form-control" id="info" rows="3"></textarea>
+      </div>
+      <div class="mb-3">
+        <input type="checkbox" name="checkbox" class="form-check-input" id="exampleCheck1">
+        <label class="form-check-label" for="exampleCheck1">Dong y dieu khoan</label>
+      </div>
+      <div class="mb-3">
+        <input type="checkbox" name="checkbox" class="form-check-input" id="exampleCheck">
+        <label class="form-check-label" for="exampleCheck1">Nhan mail tin tuc</label>
+      </div>
+      <div class="mb-3">
+        <button id="submit"  type="button" class="btn btn-primary disabled">Register</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
+  $(function() {
+    function checkety() {
+      var birthday = $("#birthday").val();
+      var address = $("#text-address").val();
+      var info = $("#info").val();
+      // name
+      const regexName = new RegExp(/[^0-9A-Za-z _-]/, 'g');
+      var name = $("#name").val();
+      var name1 = name.match(regexName);
+      // email
+      const regexEmail = new RegExp(/[^0-9A-Za-z_.-]+@/, 'g');
+      var email = $("#email").val();
+      var checkmail = email.split("");
+      var error = checkmail.indexOf("@");
+      var email1 = (email.match(regexEmail) || error == -1 || checkmail.shift() == "@" || checkmail.pop() == "@");
+      // phone
+      const regexPhone = new RegExp(/[^0-9]/, 'g');
+      var phone = $("#phone").val();
+      var numberPhone = phone.split("");
+      var phone1 = (phone.match(regexPhone) || numberPhone.shift() != "0" || numberPhone.length < 9);
+      var checkbox1 = $('#exampleCheck1').is(":checked");
+      if (name != '' && !name1 && email != '' && !email1 && phone != '' && !phone1 && address != '' && birthday != '' && checkbox1 == true ) {
```

Comment:
```suggestion
      if (name != '' && !name1 && email != '' && !email1 && phone != '' && !phone1 && address != '' && birthday != '' && checkbox1) {
```
true là boolean r, ko cần so sánh nữa